### PR TITLE
mark void dev testing fix and reverted submission and documents renam…

### PR DESCRIPTION
**Pull request by nitish-beehyv**
mark void dev testing fix and reverted submission and documents renaming conflicts

## Summary by CodeRabbit

- **New Features**
	- Enhanced functionality for evidence marking and submission voiding.
	- Introduced new validation methods for input and date constraints.
	- Updated tab labels for improved clarity in the user interface.
	- Added conditional rendering for submission status messages.
	- Improved handling of navigation and submission states.

- **Bug Fixes**
	- Improved error handling and user feedback during evidence marking and voiding processes.
	- Adjusted document upload handling to ensure correct status checks.

- **Style**
	- Updated button labels for consistency in terminology across the application.
	- Enhanced visual feedback for modal actions based on specific conditions.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 1]
```diff
@@ -383,45 +385,52 @@
...
+        config: {
+          enable: true,
+        },
+      },
+      {
+        onSuccess,
+        onError,
+      }
+    );
   };
```
[the remark this answers is not in the document]
Comment (TusharKatiyar2002) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:426]: Plz do the needful

**Review by TusharKatiyar2002 — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/HearingSideCard.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ const HearingSideCard = ({ hearingId, caseId, filingNumber }) => {
             onClick={() => handleNavigate("dristi/home/view-case", { tab: "Orders" })}
           />
           <Button
-            label={t("SUBMISSIONS")}
+            label={t("APPLICATIONS")}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/HearingSideCard.js:43]: Checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/EvidenceModal.js [hunk 3]
```diff
@@ -866,15 +870,15 @@ const EvidenceModal = ({
...
+                {((modalType === "Submissions" &&
                   [
                     SubmissionWorkflowState.PENDINGAPPROVAL,
                     SubmissionWorkflowState.PENDINGREVIEW,
                     SubmissionWorkflowState.PENDINGRESPONSE,
                     SubmissionWorkflowState.COMPLETED,
                     SubmissionWorkflowState.REJECTED,
                   ].includes(applicationStatus)) ||
-                  modalType === "Filings") && (
+                  modalType === "Documents") && (
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/EvidenceModal.js:881]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/VoidSubmissionBody.js [hunk 4]
```diff
@@ -17,24 +17,37 @@
...
+          {t("DOCUMENT_WILL_BE_PART")}
+        </p>
+      )}
       <LabelFieldPair className="case-label-field-pair">
-        <CardLabel className="case-input-label">{`${t("REASON_FOR_NOT_CONSIDERATION")}`}</CardLabel>
+        {("mark_as_void" === documentSubmission?.[0]?.itemType || "unmark_void_submission" === documentSubmission[0].itemType) && (
+          <CardLabel className="case-input-label">
+            {"unmark_void_submission" === documentSubmission[0].itemType ? t("ORIGINAL_REASON_FOR_VOIDING") : `${t("REASON_FOR_NOT_CONSIDERATION")}`}
+          </CardLabel>
+        )}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/VoidSubmissionBody.js:35]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCasesConfig.js [hunk 5]
```diff
@@ -429,7 +429,7 @@ export const TabSearchconfig = {
       },
     },
     {
-      label: "Filings",
+      label: "Documents",
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCasesConfig.js:432]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/VoidSubmissionBody.js [hunk 6]
```diff
@@ -17,24 +17,37 @@ const VoidSubmissionBody = ({ t, documentSubmission, setVoidReason, voidReason,
...
+      {"mark_as_void" === documentSubmission?.[0]?.itemType && (
         <p className="void-submission-message" style={pStyle}>
           {t("MARK_VOID_SUBMISSION_MESSAGE")}
         </p>
       )}
+      {documentSubmission[0].itemType === "unmark_void_submission" && (
+        <p className="void-submission-message" style={pStyle}>
+          {t("DOCUMENT_WILL_BE_PART")}
+        </p>
+      )}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/VoidSubmissionBody.js:29]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionDocuments.js [hunk 7]
```diff
@@ -259,9 +259,9 @@ const SubmissionDocuments = ({ path }) => {
         const documentsFile =
           signedDocumentUploadedID !== "" || localStorageID
             ? {
-              documentType: "SIGNED",
-              fileStore: signedDocumentUploadedID || localStorageID,
-            }
+                documentType: "SIGNED",
+                fileStore: signedDocumentUploadedID || localStorageID,
+              }
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionDocuments.js:264]: checked, was only formatting change

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 8]
```diff
@@ -383,45 +385,52 @@ const AdmittedCases = () => {
...
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:426]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 9]
```diff
@@ -1020,7 +1045,7 @@
           console.error("Error while creating order", error);
           showToast({ isError: true, message: "ORDER_CREATION_FAILED" });
         });
-    } catch (error) { }
+    } catch (error) {}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:1048]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 10]
```diff
@@ -1537,7 +1563,7 @@
               tab: "Orders",
             });
           })
-          .catch((err) => { });
+          .catch((err) => {});
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:1566]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:426]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js [hunk 11]
```diff
@@ -1067,7 +1067,7 @@ export const UICustomizations = {
       return [
         ...((userInfo.roles.map((role) => role.code).includes("JUDGE_ROLE") ||
           userInfo.roles.map((role) => role.code).includes("COURT_ROOM_MANAGER")) &&
-        row.status === "SUBMITTED" &&
+        !row.isVoid &&
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js:1070]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/DocumentModal.js [hunk 12]
```diff
@@ -107,6 +107,9 @@ const DocumentModal = ({ config, setShow, currentStep, documentStyle = {} }) =>
       popUpStyleMain={documentStyle}
       isDisabled={isDisabled}
       textStyle={submitButtonTextStyle}
+      style={
+        config?.isStepperModal ? (config.steps[step]?.actionSaveLableType === "WARNING" ? { backgroundColor: "#BB2C2F", border: "none" } : {}) : {}
+      }
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/DocumentModal.js:112]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js [hunk 13]
```diff
@@ -1067,14 +1067,14 @@ export const UICustomizations = {
       return [
         ...((userInfo.roles.map((role) => role.code).includes("JUDGE_ROLE") ||
           userInfo.roles.map((role) => role.code).includes("COURT_ROOM_MANAGER")) &&
-        row.status === "SUBMITTED" &&
-        !Boolean(row?.isVoid)
+        !row.isVoid &&
+        row?.filingType === "DIRECT"
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js:1071]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 14]
```diff
@@ -1523,8 +1552,9 @@ const AdmittedCases = () => {
                   code: "INITIATING_RESCHEDULING_OF_HEARING_DATE",
                   name: "ORDER_TYPE_INITIATING_RESCHEDULING_OF_HEARING_DATE",
                 },
-                originalHearingDate: `${date.getFullYear()}-${date.getMonth() < 9 ? `0${date.getMonth() + 1}` : date.getMonth() + 1}-${date.getDate() < 10 ? `0${date.getDate()}` : date.getDate()
-                  }`,
+                originalHearingDate: `${date.getFullYear()}-${date.getMonth() < 9 ? `0${date.getMonth() + 1}` : date.getMonth() + 1}-${
+                  date.getDate() < 10 ? `0${date.getDate()}` : date.getDate()
+                }`,
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:1557]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/EvidenceModal.js [hunk 15]
```diff
@@ -807,10 +809,14 @@ const EvidenceModal = ({
...
+                modalType === "Documents"
+                  ? documentSubmission?.[0]?.artifactList?.isEvidence
+                    ? "Accepeted"
+                    : "Action Pending"
+                  : t(applicationStatus)
               }
-              showStatus={modalType === "Filings" ? false : true}
-              isStatusRed={modalType === "Filings" ? !documentSubmission?.[0]?.artifactList?.isEvidence : applicationStatus}
+              showStatus={modalType === "Documents" ? false : true}
+              isStatusRed={modalType === "Documents" ? !documentSubmission?.[0]?.artifactList?.isEvidence : applicationStatus}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/EvidenceModal.js:819]: checked

**Review by TusharKatiyar2002 — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js [hunk 16]
```diff
@@ -1052,7 +1052,7 @@ export const UICustomizations = {
           return <Evidence userRoles={userRoles} rowData={row} colData={column} t={t} />;
         case "STATUS":
           //Need to change the shade as per the value
-          return row?.status ? <CustomChip text={t(row?.status)} shade={"green"} /> : "";
+          return row?.isVoid ? "VOID" : row?.status ? <CustomChip text={t(row?.status)} shade={"green"} /> : "";
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js:1055]: checked

**Pull request merged**